Source/Editor/DualityEditor/Utility/EditorHelper.cs

```diff
@@ -40,6 +40,17 @@ public static class EditorHelper
 		private static bool isJitDebuggerAvailable;
 		private static VisualStudioEdition vsEdition;
 
+		/// <summary>
+		/// The path to the *.sln solution file. Will return null if the solution file has not yet been created.
+		/// </summary>
+		public static string SourceCodeSolutionFilePath
+		{
+			get
+			{
+				return Directory.EnumerateFiles(SourceCodeDirectory, "*.sln", SearchOption.AllDirectories).FirstOrDefault();
```

Comment:
Should not return null if no solution exists! It's just a path, no statement made about existence. Instead, return the default path at which the .sln will be created.

Reply:
In a second step, will need to replace the newly introduced `== null` checks elsewhere in code with file existence checks to match.